src/components/post-form/content-form.tsx

```diff
@@ -0,0 +1,201 @@
+import { useContext } from "react";
+import { useForm } from "react-hook-form";
+import { MdAdd, MdRemove } from "react-icons/md";
+
+import { PosterContext } from "@/context/poster";
+
+import { Button } from "../ui/button";
+import { DatePicker } from "../ui/date-picker";
+import { Input } from "../ui/input";
```

Comment:
This kind of components are imported from ts normal library since I cannot find them on the ui folder, right?

Reply:
Those components come from the `@/components/ui` folder as you can see in the absolute paths.